docs/tutorials/services/navigate-with-rover-base.md

```diff
@@ -36,12 +36,21 @@ weight: 5
 # SMEs: Ray Bjorkman, Fahmina
 ---
 
-One key feature of Viam is [the navigation service](/mobility/navigation/), the stateful definition of Viam's [motion service](/mobility/motion/).
+The [navigation service](/mobility/navigation/) is the stateful definition of the [motion service](/mobility/motion/).
```

Comment:
[opt] Since many users don't know what "stateful definition" means, could change, but that'd be out of diff

Reply:
I don't know what else to use - do you?